CHANGELOG.rst

```diff
@@ -401,7 +416,6 @@ New Modules
 - iosxr_command - Run commands on remote devices running Cisco IOS XR
 - iosxr_config - Manage Cisco IOS XR configuration sections
 - iosxr_facts - Get facts about iosxr devices.
-- iosxr_interface - (deprecated, removed after 2022-06-01) Manage Interface on Cisco IOS XR network devices
```

Comment:
Why go back in history to remove this?

Reply:
This should still be in the changelog. Just because we removed it doesn't mean we didn't add it in the past.